templates/user/dashboard/feeds.tmpl

```diff
@@ -11,75 +11,75 @@
 					{{else}}
 						{{.ShortActUserName}}
 					{{end}}
-					{{if eq .GetOpType 1}}
+					{{if .GetOpType.IsActionCreateRepo}}
```

Comment:
What's the difference of `eq .GetOpType.String "create_repo"` or `.GetOpType.IsAction "create_repo"` ?

IMO these `IsXxx` functions only increases code lines, but doesn't bring real benefit. eg: the code is still executed dynamically and no static check.

Reply:
I guess it can be `func IsAction(actions ...string) bool`, then the template `or` can also be eliminated

Reply:
Sounds great, thanks so much for the advice!

Reply:
Done [`17fea19` (#26956)](https://github.com/go-gitea/gitea/pull/26956/commits/17fea1955c935800306ad6b6b177a4c7757a2218)